src/components/main-nav/main-nav.js

```diff
@@ -6,5 +6,43 @@ export default class MainNav {
   constructor (node) {
     this.node = document.querySelector(node)
     this.node.innerHTML = template(mainNavData)
+    this.items = this.node.querySelectorAll('.menu__item')
+    this.links = this.node.querySelectorAll('.menu__link')
+    this.toggle = document.querySelector('.header__button')
+    this.nav = document.querySelector('.nav')
+    this.deploySubmenuEvent()
+    this.toggleMenuEvent()
+  }
+
+  static get states () {
+    return {
+      menuLinkAnimated: 'menu__link--animated'
+    }
+  }
+
+  deploySubmenuEvent () {
+    this.items.forEach((item, index, array) => {
+      item.addEventListener('click', e => {
+        this.resetClass(index)
+        item.querySelector('.submenu').classList.toggle('submenu--show')
```

Comment:
I would say that `submenu--show` should also be part of the `states` static property.